DMX.Gatekeeper.Api.Tests.Unit/Services/Foundations/Labs/LabServiceTests.cs

```diff
@@ -65,7 +65,7 @@ public static TheoryData CriticalDependencyException()
         private static int GetRandomNumber() =>
             new IntRange(min: 2, max: 10).GetValue();
 
-        private static Filler<Lab> CreateLabFiller() => 
+        private static Filler<Lab> CreateLabFiller() =>
```

Comment:
I need to find a trailing whitespace remover extension :)

Reply:
It's out of the box. just do this:

![Cleanup](https://user-images.githubusercontent.com/1453985/172029689-c5e23b03-d111-48bc-8fd3-91a805ad7151.gif)

Reply:
I mean on file save :) I tend to forget to manually do it (which is why I had Code Maid)